### PR TITLE
PMM-7250 rename GetXXXCluster to GetXXXClusterCredentials

### PR DESCRIPTION
**Pull request by jprukner**
https://jira.percona.com/browse/PMM-7250

To make it clear, the API call should contain word credentials because it is actually why we call it.

### CODE REVIEW

File: api/managementpb/dbaas/psmdb_cluster.proto [hunk 1]
```diff
@@ -108,14 +108,6 @@ message GetPSMDBClusterResponse {
...
     string replicaset = 5;
   }
-  // Cluster state.
-  PSMDBClusterState state = 1;
-  // Currently running operation, if any.
-  RunningOperation operation = 2;
-  // Cluster parameters.
-  PSMDBClusterParams params = 3;
-  // Connection credentials.
-  PSMDBCredentials connection_credentials = 4;
```
Comment (BupycHuk) [inline api/managementpb/dbaas/psmdb_cluster.proto:118]: let's keep credentials :)

**Review by BupycHuk — CHANGES_REQUESTED** [review]

File: api/managementpb/dbaas/psmdb_cluster.proto (hunk 1 above)
Comment (jprukner) [reply api/managementpb/dbaas/psmdb_cluster.proto:118]: They are here now, I just moved the PSMDBCredentials outside of the response message. Or did you mean keep the message definition inside the response message definition?

Comment (BupycHuk) [reply api/managementpb/dbaas/psmdb_cluster.proto:118]: Yeah, let's keep it inside since we don't use `PSMDBCredentials` anywhere else.

Comment (jprukner) [reply api/managementpb/dbaas/psmdb_cluster.proto:118]: done

**Review by BupycHuk — APPROVED** [review]

**Review by JiriCtvrtka — APPROVED** [review]

**Review by AlekSi — APPROVED** [review]

**Comment by AlekSi** [comment]
@BupycHuk Shouldn't it go into `release/2.15`?

**Pull request merged**